### ISSUE TITLE
Documentation should reflect new clang-format enforced style

### ISSUE THREAD

**Issue by dzenanz** [InsightSoftwareConsortium/ITK#1270]
#1191 changed the code style, documentation should reflect this change, both style-wise and process-wise.

**Comment by hjmjohnson** [comment]
```
ITKSoftwareGuide/SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex
```

**Comment by jhlegarreta** [comment]
Isn't this issue more appropriate for the ITK SW Guide?
https://github.com/InsightSoftwareConsortium/ITKSoftwareGuide/issues/133

**Comment by dzenanz** [comment]
I forgot we have a separate repository for documentation, but there is documentation outside of software guide, e.g. https://github.com/InsightSoftwareConsortium/ITK/blob/master/Documentation/GitHelp.md.

**Comment by jhlegarreta** [comment]
True, but that folder does not contain any style guide-related contents if I am not mistaken, right?

**Comment by dzenanz** [comment]
There is now a git hook which is being run by default, I believe.
